Packages/Application/Neos.NeosIo.Event/Resources/Private/Templates/TypoScriptObjects/FooterContent.html

```diff
@@ -11,14 +11,25 @@ <h3 class="h1 typo3-neos-alignment-center">Organizer</h3>
 		</f:if>
 		<div class="gi u-w1/1 u-wm6/12 u-wl6/12">
 			<f:if condition="{organizers.properties.logo}">
-				<figure>
-					<media:image asset="{organizers.properties.logo}" title="{organizers.properties.title}" alt="{organizers.properties.title}" maximumWidth="700" maximumHeight="500" allowUpScaling="false" />
-				</figure>
+				<f:if condition="{organizers.properties.website}">
```

Comment:
Yeah ok. But we should fix this as this some kind of "How to do things with Neos correct", and this is not what ppl should do :D